public/scripts/gif.js

```diff
@@ -0,0 +1,65 @@
+var imagesData = [];
+var imageLinks = [];
+
+function getImageArray(data){
+
+	imagesData = [];
+	$("#gif #gifContent #gifApp").empty();
+	$.each(data, function (index, obj) {
+		if($(obj.event_html).attr("src").indexOf("notcaptured") < 0){
+		imagesData.push(obj);
+		}
+	});
+
+	for(var i = 0; i<imagesData.length; i++)
+	{
+		imageLinks[i] = $(imagesData[i].event_html).attr('src');
+		console.log(imageLinks[i]);
+	}
+
+	var interval = document.getElementById("interval").value;
+
+	create(imageLinks, interval);
+
+	document.getElementById("gifButton").addEventListener("click", updateGif);
+}
+
+function updateGif(){
+	$("#gifApp").empty();
+	var interval = document.getElementById("interval").value;
+	create(imageLinks, interval);
+}
+	
+
+
+
+function create(image, int)

```

Comment:
```suggestion
function createGif(image, interval)
```

The function name `create` is too generic and can easily collide with something else if leaked into the global name space. There is a function defined as `updateGif` already, so following that pattern this should be named accordingly.

The second parameter's (i.e., `interval`) short name has an unfortunate collision with a more common keyword `int` (for integer). Hence, replacing it to avoid confusion.